tests/dialects/test_vector.py

```diff
@@ -0,0 +1,79 @@
+from xdsl.dialects.builtin import i32, IntegerType, IndexType, VectorType
+from xdsl.dialects.memref import MemRefType
+from xdsl.dialects.vector import Load, Store
+from xdsl.ir import SSAValue, OpResult
+
+
+def test_vectorType():
+    vec1 = VectorType.from_type_and_list(i32)
+
+    assert vec1.get_num_dims() == 1
+    assert vec1.get_shape() == [1]
+    assert vec1.element_type is i32
+
+    vec2 = VectorType.from_type_and_list(i32, [3, 3, 3])
```

Comment:
Please split these into three separate tests, one for eachvector you create. IMO, this will give better insights if tests fail.